### PR TITLE
[Docs] First version of dashboard docs

### PR DESCRIPTION
**Pull request by lingyielia**
## Notice

- [x] I acknowledge and agree that, by checking this box and clicking "Submit Pull Request":

  - I submit this contribution under the [Apache 2.0 license](https://www.apache.org/licenses/LICENSE-2.0.txt) and represent that I am entitled to do so on behalf of myself, my employer, or relevant third parties, as applicable.
  - I certify that (a) this contribution is my original creation and / or (b) to the extent it is not my original creation, I am authorized to submit this contribution on behalf of the original creator(s) or their licensees.
  - I certify that the use of this contribution as authorized by the Apache 2.0 license does not violate the intellectual property rights of anyone else.
  - I have not referenced individuals, products or companies in any commits, directly or indirectly.
  - I have not added data or restricted code in any commits, directly or indirectly.

### CODE REVIEW

**Review by maxschulz-COL — COMMENTED** [review]
Just as a heads up: the docs build fails which is annoying when reviewing - could you check that this works in the future? 🙏

**Comment by stichbury** [comment]
> Just as a heads up: the docs build fails which is annoying when reviewing - could you check that this works in the future? 🙏

Snap! I've just fixed this -- see my commit

File: vizro-ai/docs/pages/user-guides/dashboard-iterate-and-deploy.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+## Setup
+```py
+%pip install vizro_ai
+
+from vizro import Vizro
+from vizro_ai import VizroAI
+
+vizro_ai = VizroAI()
+```
```
Comment (maxschulz-COL) [inline vizro-ai/docs/pages/user-guides/dashboard-iterate-and-deploy.md:15]: No need for this

File: vizro-ai/docs/pages/user-guides/dashboard-iterate-and-deploy.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+2. Table:
+   - Position: Right of the card
+   - Size: Takes up the remaining 3/4 of the page width
+   - Content: Display the Tips dataset
+"""
+```
+
+## Generate and launch the dashboard
+```py
+dashboard_elements = vizro_ai.dashboard([df], user_question, return_elements=True)
```
Comment (maxschulz-COL) [inline vizro-ai/docs/pages/user-guides/dashboard-iterate-and-deploy.md:40]: Would combine into one code snippet that goes from imports to object to render in one go. This should be the first example, then we can discuss refinements, code snippets etc

File: vizro-ai/docs/pages/user-guides/dashboard-models.md
Comment (maxschulz-COL) [inline vizro-ai/docs/pages/user-guides/dashboard-models.md:1]: maybe make this a table actually?

**Review by maxschulz-COL — COMMENTED** [review]
🚀  Lots of chaotic comments, hope the overall gist is clear. Mainly trying to more closely follow our diataxis framework again, and ensuring the how to guide start with simple complete example, then discusses options

File: vizro-ai/docs/pages/user-guides/dashboard-iterate-and-deploy.md
[the remark this answers is not in the document]
Comment (stichbury) [reply vizro-ai/docs/pages/user-guides/dashboard-iterate-and-deploy.md:1]: This is for structuring the How-to section, and yes, I pretty much agree. I think we may need a MODELS section so we keep "Customize models" and "Choose the right model" together, but with clear distinction when content applies specifically to dashboards, or to charting, or to any task.

**Pull request merged**